src/ray/rpc/gcs_server/gcs_rpc_client.h

```diff
@@ -66,11 +72,25 @@ class Executor {
         executor->Retry();                                                             \
       }                                                                                \
     };                                                                                 \
-    auto operation = [request, operation_callback](GcsRpcClient *gcs_rpc_client) {     \
+    auto operation = [request, operation_callback,                                     \
+                      timeout_ms](GcsRpcClient *gcs_rpc_client) {                      \
       RAY_UNUSED(INVOKE_RPC_CALL(SERVICE, METHOD, request, operation_callback,         \
-                                 gcs_rpc_client->grpc_client, method_timeout_ms));     \
+                                 gcs_rpc_client->grpc_client, timeout_ms));            \
     };                                                                                 \
     executor->Execute(operation);                                                      \
+  }                                                                                    \
+                                                                                       \
+  ray::Status Sync_##METHOD(const METHOD##Request &request, METHOD##Reply *reply_in,   \
```

Comment:
nit: `Sync##Method?`
but it's up to you.

Reply:
Kind of followed your STATS_.. (I liked that one because it is clear, it was generated) 

I don’t have much preference. Is there any convention you are aware of here?

Reply:
I decided to use SYNC_[METHOD] following STATS_[stats_name]

Reply:
I use `STATS_` because of `FLAGS_` 😅

`METHOD` is also generated right?
Hmm, maybe `ASYNC_##METHOD` and `SYNC_##METHOD` if we follow this one. (I don't mean to update it here).